Doc updates for MSTest (#43786)
dotnet/docs: docs/core/testing/unit-testing-platform-config.md

When using [Microsoft.Testing.Platform.MSBuild](https://www.nuget.org/packages/Microsoft.Testing.Platform.MSBuild), you can simply create a *testconfig.json* file that will be automatically renamed to *[appname].testconfig.json* and moved to the output directory of the test project.

> Starting with Microsoft.Testing.Platform 1.5, the command-line argument `--config-file` can be used to specify the path to the *testconfig.json*. This takes precedence over the *[appname].testconfig.json* file.

Reviewer: ```suggestion
Starting with Microsoft.Testing.Platform 1.5, you can use the command-line argument `--config-file` to specify the path to the *testconfig.json*. This file takes precedence over the *[appname].testconfig.json* file.
```

> [!NOTE]
> Environment variables take precedence over configuration settings in the *testconfig.json* file.

> ### `TESTINGPLATFORM_UI_LANGUAGE` environment variable

Author: Is this the right doc page for documenting this? @Evangelink

Author: @Evangelink This is the docs for `DOTNET_CLI_UI_LANGUAGE`
https://github.com/dotnet/docs/blob/3208829cb687c603ab97762f8cfad9aa12dd3c4c/docs/core/tools/dotnet-environment-variables.md?plain=1#L335-L337

Reviewer 2: We can copy the first part:

```
Sets the language of the CLI UI using a locale value such as `en-us`. The supported values are the same as for Visual Studio. For more information, see the section on changing the installer language in the [Visual Studio installation documentation](/visualstudio/install/install-visual-studio).
```

In the `dotnet test` integration doc we could link to this part of the CLI doc.

> Starting with Microsoft.Testing.Platform 1.5, this environment variable sets the language of the platform for displaying messages and logs using a locale value such as `en-us`. This takes precedence over the Visual Studio and .NET SDK languages. The supported values are the same as for Visual Studio. For more information, see the section on changing the installer language in the [Visual Studio installation documentation](/visualstudio/install/install-visual-studio).

Reviewer: ```suggestion
Starting with Microsoft.Testing.Platform 1.5, this environment variable sets the language of the platform for displaying messages and logs using a locale value such as `en-us`. This language takes precedence over the Visual Studio and .NET SDK languages. The supported values are the same as for Visual Studio. For more information, see the section on changing the installer language in the [Visual Studio installation documentation](/visualstudio/install/install-visual-studio).
```
